src/Routes/Login.js

```diff
@@ -1,51 +1,30 @@
 import React from "react";
-import AppButton from "../UI/Button";
-import GenericInput from "../UI/Input";
-import Modal from "../UI/Modal";
+import styled from "styled-components";
 import NavBar from "../UI/NavBar";
-import { UserOutlined, KeyOutlined } from "@ant-design/icons";
-
-const iconStyling = {
-  color: "#c1c1c1",
-  fontSize: "18px",
-  marginLeft: "8px",
-};
-
+import LoginModal from "../UI/LoginModal";
+import BlurDiv from "../UI/BlurDiv";
+import backgroundImage from "../StarsBG.jpg";
```

Comment:
aint this background a bit pixelated.
is the ratio that you are using working with different screens

Reply:
It is blurred on purpose. We can change the blurry-ness with a prop. the actual image is in 4k so it should work on most displays.